README.md

```diff
@@ -126,25 +223,26 @@ You should see a list of all the files within your working directory.
 
 The command `ls` stands for "**l**i**s**t".
 
-We can use flags on most unix commands to give more specific instructions. Most programs also accept
-flags, or options for execution.
+We can use flags on most unix commands to give more specific instructions. Most
+programs also accept flags, or options for execution.
 
-A flag is denotated by a `-` ("dash"). **Note:** *In some programs, options are passed directly to the
-command and not via flags.*
+A flag is denotated by a `-` ("dash"). **Note:** *In some programs, options are
```

Comment:
Break the note into a separate para. and use `> ` to set it off in Markdown. Join this first sentence to prev. para.